### DISCUSSION TITLE
Flame Shock recommendation w/ enhance

### DISCUSSION THREAD

**Discussion by basalt51** [Hekili/hekili#1413] [Q&A]
Hekili seems to randomly recommend Flame Shock even with 15-20 seconds left. My talents are 1231231, also did it with 3231231. I'm not sure if its not recognizing that lava lash refreshes flame shock now or if this is only happening during rare periods of nothing else to cast? Sometimes it's recommended for a fraction of a second and suddenly disappears just before I push the button and something else is recommended.  Other times it seems everything recommend after flame shock is on cooldown.

Just wondering if this is new an "issue" or simply the only thing to cast until other things come off cooldown.

I will say, my Shaman sims at 7900 on the test dummy, but following the recommendations as closely as I can, I'm doing 8200-8400!

**Comment by Hekili** [comment]
Yeah, you can look at the priority; there is a low priority Flame Shock when you basically have nothing else to do.  That "nothing else to do" period of time might be a very short fraction of a second, then you have something else to press.

**Reply by Jroy21** [reply]
I'm running into the same issue and would suggest Flame Shock when already active is a DPS lost even if you have nothing else to press for a fraction of a GCD. Because humans are using this the distraction from this Flame Shock popping up breaks the flow. If I knew how to modify the script myself I would implement this simple condition to never recommend Flame Shock if the target already has it active.

**Reply by Hekili** [reply]
> Flame Shock when already active is a DPS lost even if you have nothing else to press for a fraction of a GCD

But it isn't.

> Because humans are using this the distraction from this Flame Shock popping up breaks the flow.

You don't have to press it.  You can wait for something else to be recommended.  It's just not optimal to do that.

> If I knew how to modify the script myself I would implement this simple condition to never recommend Flame Shock if the target already has it active.

You don't have to do that; you can just toggle the setting that I directed you to in your ticket.
